### DISCUSSION TITLE
other/Info

### DISCUSSION THREAD

**Discussion by giscus** [TakagisanArchiveRepos/VuePress-TakagiWiki#4] [评论区（Gistus General）]
# other/Info

内容引用相关 本站部分内容参考 [Fandom社区]( https://karakai-jouzu-no-takagi-san.fandom.com/zh/wiki/擅长捉弄的高木同学) 与 维基百科 (https://zh.m.wikipedia.org/wiki/%E6%93%85%E9%95%B7%E6%8D%89%E5%BC%84%E4%BA%...

https://wiki.takagi3.cn/other/Info.html

**Comment by PaloMiku** [comment]
一只鬆：评论功能测试
